app/models/case_log.rb

```diff
@@ -48,15 +48,16 @@ def form
     FormHandler.instance.get_form(form_name) || FormHandler.instance.forms.first.second
   end
 
+  def get_collection_start_year
```

Comment:
Style thing but I usually prefer to name the method the thing it returns and use it directly. So this would become

```
def collection_start_year
    window_end_date = Time.zone.local(startdate.year, 4, 1)
    startdate < window_end_date ? startdate.year - 1 : startdate.year
end

def form_name
  "#{collection_start_year}_#{collection_start_year + 1}"
end
```

The `get`, `set` thing reminds me of java... 

Just personal preference though so feel free to ignore entirely!

Reply:
Nah, that makes sense!